GUI/MainModList.cs

```diff
@@ -273,6 +274,14 @@ private void _UpdateModsList(bool repo_updated, IEnumerable<ModChange> mc)
 
             UpdateFilters(this);
 
+            // Hide update and replacement columns if not needed.
+            // Write it to the configuration, else they are hidden agian after a filter change.
+            // After the update / replacement, they are hidden again.
+            ModList.Columns[1].Visible = has_any_updates;
+            configuration.VisibleColumns[1] = has_any_updates;
```

Comment:
Configuration probably should be modified here if we're no longer including update/replace in the list.